Client/Frontend/Widgets/BlurTableViewController.swift

```diff
@@ -0,0 +1,182 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import Storage
+
+class BlurTableViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
+    var profile: Profile!
```

Comment:
You should try to abstract away all properties from this class not related to showing a menu. The BlurTableViewAction's blocks should contain the information needed to perform actions